src/js/data-model.jsx

```diff
@@ -1,82 +1,153 @@
+            var table = _.find(this.props.tables, function(f) { return f.name === self.props.activeTable; });
+            if(self.props.activeField) {
+                description = _.find(table.fields, function(f) { return f.name === self.props.activeField; }).description;
+                self.state.collapsedStatus[table.name] = false; // if a field is selected, the table shouldn't be collapsed
+            }
+            else {
+                description = table.description;
+            }
+        }
+        
+        var tables = _.map(this.props.tables, function(table) {
+            var isActiveTable = (table.name === self.props.activeTable);
             return (
-                <li key={item.ident} className={className}>
-                    <a href={url}>{item.label}</a>
+                <li key={table.name}>
+                    <Table model={self.props.name} 
+                       version={self.props.version}
+                       name={table.name}
+                       fields={table.fields}
+                       isActive = {isActiveTable} 
+                       activeField = {isActiveTable? self.props.activeField : ''}
+                       isCollapsed = {self.isTableCollapsed(table.name)}
+                       handleClick = {self.toggleCollapsed(table.name)}/>
                 </li>
             );
         });
-
-        var details, entity, object, log;
-
-        if (this.props.resource) {
-            entity = entities.index[this.props.resource];
-
-            if (entity) {
-                object = <DataModelObject {...entity} />;
-
-                details = (
-                    <div>
-                        {object}
-
-                        <hr />
-
-                        <h4>Log</h4>
-
-                        <FactLog facts={entity.facts} comparator={Comparators.time} />
-                    </div>
-                );
-            }
-        }
-
+        
+        var dataDictElement = this.props.siteComments ?
+            <li key='data-dict'>
+                <CollapsibleText title={'Site Comments [Implementation Status: ' + this.props.siteComments.status + ']'} 
+                                  content={this.props.siteComments.comment}/>
+            </li>
+            : null;
+
+        var descriptionElement = description === '' ? null :
+            <li key='description'>
+                <CollapsibleText title='Description' content={description}/>
+            </li>;
+
+        var etlConventionsElement = this.state.etlConventions ?
+            <li key='etl'>
+                <CollapsibleText title='ETL Conventions' content={this.state.etlConventions}/>
+            </li>
+            : null;
+
+        var dqaElement = (this.props.dqa && this.props.dqa!=={} &&
+                           this.props.activeTable!='' && this.props.activeField!='' &&
+                           (Object.keys(this.props.dqa).indexOf(this.props.activeTable)>=0) &&
```

Comment:
Also reading more below, it seems like the value of `this.props.dqa[this.props.activeTable]` is used several times. It would be more clear to create a local variable and then test it in the `if` block when deciding to the set the element.

```javascript
var dqaTable = this.props.dqa && this.props.dqa[this.props.activeTable],
    dqaField = dqaTable && dqaTable.fields[this.props.activeField];

if (dqaTable !== undefined) {
    ...
}
```

Since we know there are no tables in the `dqa` object with the empty string (nor fields in a table), we can just rely on the existence of the entry.